packages/evm/contracts/treasury/ProtocolTreasury.sol

```diff
@@ -55,17 +55,17 @@ contract ProtocolTreasury is ITreasury, ControlledRecoverable, SignaturesValidat
     * @param _to Address of the recipient that will receive the tokens
     * @param _amount Amount of tokens to be withdrawn from the sender
     */
-    function withdraw(IERC20 _token, address _from, address _to, uint256 _amount) external authenticate(_from) {
+    function withdraw(IERC20 _token, address _from, address _to, uint256 _amount) external authenticateSender(_from) {
         _withdraw(_token, _from, _to, _amount);
     }
 
     /**
-    * @notice Withdraw all the tokens from `_from` to themself
+    * @notice Withdraw all the tokens from `_from` to themselves
     * @param _token ERC20 token to be withdrawn
     * @param _from Address withdrawing the tokens from and where the tokens will be transferred
     */
     function withdrawAll(IERC20 _token, address _from) external {
```

Comment:
From our discussion yesterday, perhaps we can remove this function altogether? Since we can re-introduce it at a later time (and perhaps, the config for withdrawals can also be removed since this sort of functionality would be pushed outside).